docs/Installation.md

```diff
@@ -118,17 +122,50 @@ sudo install --mode=755 ./backend_config.ini-postgresql /etc/zonemaster/backend_
 sudo install --mode=755 ./zm-centos.sh-postgresql /etc/init.d/zm-centos.sh
 mkdir "$HOME/logs"
 ```
+Get the appropriate PostgreSQL binary package
 
-Install, configure and start database engine (and Perl bindings):
+```sh
+sudo rpm -iUvh https://yum.postgresql.org/9.3/redhat/rhel-7-x86_64/pgdg-centos93-9.3-3.noarch.rpm
+```
 
->
-> At this time there is no instruction for using PostgreSQL on CentOS.
->
-> [Sandoche! I think we should be able to support PostgreSQL!
-> Can you try to find PostgreSQL package plus perl lib?]
->
+Install the PostgreSQL packages:
 
-Initialize the database:
+```sh
+sudo yum -y install postgresql93 postgresql93-server postgresql93-contrib postgresql93-libs postgresql93-devel perl-DBD-Pg
+```
+
+# To enable PostgreSQL from boot
+
+```sh
+sudo systemctl enable postgresql-9.3
+```
+
+# Initialise PostgreSQL
+
+```sh
+sudo /usr/pgsql-9.3/bin/postgresql93-setup initdb
+```
+
+# Configure:
+
+```sh
+# In the below file modify all instances of "ident" to "md5"
+sudo vi /var/lib/pgsql/9.3/data/pg_hba.conf

```

Comment:
It's good practice to use `sudoedit` instead of `sudo vi`. It limits the write access to the particular file(s) given on the command line.

Reply:
ok